### DISCUSSION TITLE
How to bulk-update the front-matter date last updated field based on the modified date on disk

### DISCUSSION THREAD

**Discussion by prdsoftware** [google/docsy#568] [General]
G'day Docsy fans,

I'm wondering if there is an easy way to bulk-update the date field in the front-matter to correspond with the "Last modified" date on disk for that file?  

I have a large site, but most of the dates are not correct.  Is there a file utility that people are using for this purpose, or another clever way?

Thanks,
Rod

**Comment by narrenfrei** [comment]
There are varios dates in the Hugo front matter: https://gohugo.io/content-management/front-matter/
Which parameter do you want to show where?

As default Docsy shows the date of last modify based on git in the footer of each docs page when in the config.toml parameter `enableGitInfo = true`.

**Reply by prdsoftware** [reply]
OK, thanks - that was easy!  The "enabledGitInfo" option was set to false for me.  I changed it to true and the date showed up.

It was the "date" parameter that I was asking about.

Is there anyway to remove the github hyperlink that is now displayed next to the update date?  It's not a public repository, so there is no need to display this.

**Comment by narrenfrei** [comment]
You have to save a adapted partial page-meta-lastmod.html unter your projects layouts/partials folder
 - https://github.com/google/docsy/blob/master/layouts/partials/page-meta-lastmod.html

**Reply by prdsoftware** [reply]
Fantastic.  Thank you!
